lib/brains/views/method.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from lib.brains.client import get_client
+from lib.brains.errors import BraintreeResultError
+from lib.brains.forms import PaymentMethodForm
+from lib.brains.serializers import (
+    PaymentMethodSerializer as BraintreeSerializer)
+from lib.buyers.models import PaymentMethod
+from lib.buyers.serializers import PaymentMethodSerializer
+from lib.transactions.constants import PROVIDER_BRAINTREE
+from solitude.constants import PAYMENT_METHOD_CARD
+from solitude.logger import getLogger
+
+
+log = getLogger('s.brains')
+
```

Comment:
Now you've got me confused :) You've got one endpoint, which is a specialized endpoint for your needs that does all the work you need.